wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -589,6 +589,22 @@ def bulk_action_choices(context, app_label, model_name):
     return {"buttons": bulk_action_buttons}
 
 
+@register.inclusion_tag("wagtailadmin/shared/shared_avatar.html")
+def avatar(size=None, tooltip=None, user=None):
+    """
+    Displays a user avatar using the avatar template
+    Usage:
+    {% load wagtailadmin_tags %}
+    ...
+    {% avatar user=request.user size='small' username='JaneDoe' %}
```

Comment:
I think this is meant to be `tooltip` in the example and in the param docs below.

Also, let's put `user` first and a reminder that we want to add a `classname` param.